README.md

```diff
@@ -29,42 +29,52 @@ The module website development environment can be installed on macOS, Windows, a
 
 ---
 
-<h2 id="built-with">Built With</h2>
+## Prerequisites
 
-The following generator is built using the following software, you'll need them installed in your workspace to run properly. 
-* [Node](https://nodejs.org/en/)
-* [Gatsby](https://www.gatsbyjs.org/)
```

Comment:
We are building with Gatsby, but it's actually not required to have it installed, as it is managed as a dependency via Yarn.